kaltura-ui/src/input-helper/input-helper.component.scss

```diff
@@ -0,0 +1,48 @@
+@import '../styles/scss/theme';
+
+.kInputHelper {
+    display: inline-block;
+    vertical-align: middle;
+    margin-left: 6px;
+    position: relative;
+  
+    .kIconhelpLink {
+      color: $kGrayscale3;
+      text-decoration: none;
+      position: relative;
+      z-index: 2;
+  
+      &:hover, &.kActive {
+        color: $kSecondary;
+      }
+    }
+  }
+  
+  .kInputHelperPopup {
+    border-radius: 3px;
+    
+    /deep/ .kPopupWidget {
```

Comment:
deep is deprecated, should use ng::deep according to angular doc:

> The shadow-piercing descendant combinator is deprecated and support is being removed from major browsers and tools. As such we plan to drop support in Angular (for all 3 of /deep/, >>> and ::ng-deep). Until then ::ng-deep should be preferred for a broader compatibility with the tools.
https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep